### PR TITLE
API syntax change

### PR DESCRIPTION
**Pull request by MaxParadiz**
Instagram changed the syntax a bit for getting the media by tag, so I fixed that.

### CODE REVIEW

**Comment by marcelcaferati** [comment]
@Leastham2  Got the same problem here.

File: src/instabot.py [hunk 1]
```diff
@@ -634,11 +634,11 @@ def new_auto_mod_unfollow(self):
     def new_auto_mod_comments(self):
         if time.time() > self.next_iteration["Comments"] and self.comments_per_day != 0 \
                 and len(self.media_by_tag) > 0 \
-                and self.check_exisiting_comment(self.media_by_tag[0]['code']) == False:
+                and self.check_exisiting_comment(self.media_by_tag[0]['node']['comments_disabled']) == False:
```
Comment (pereiren) [inline src/instabot.py]: Replace comments_disabled by shortcode

Comment (marcelcaferati) [reply src/instabot.py]: It is already like that on @MaxParadiz file, but still getting the same problem

Comment (pereiren) [reply src/instabot.py]: For me it is working fine using shortcode
image

**Comment by marcelcaferati** [comment]
Yea, i edited my answear, I missunderstood what you said. Seems to be working so far, still doing some tests.

**Comment by Leastham2** [comment]
the fix is live! thank you

**Comment by GithubProgress** [comment]
This has been fixed by a different pull request .

**Pull request closed without merging**